components/locid_transform/src/fallback/mod.rs

```diff
@@ -49,9 +14,44 @@ pub use icu_provider::fallback::*;
 
 mod algorithms;
 
-/// Entry type for locale fallbacking.
+/// Implements the algorithm defined in *[UTS #35: Locale Inheritance and Matching]*.
 ///
-/// See the module-level documentation for an example.
+/// Note that this implementation performs some additional steps compared to the *UTS #35*
+/// algorithm, see *[the design doc]* for a detailed description, and [#2243](
+/// https://github.com/unicode-org/icu4x/issues/2243) to track aligment with *UTS #35*.
+///
+/// # Examples
+///
+/// ```
+/// use icu_locid::locale;
+/// use icu_locid_transform::fallback::LocaleFallbacker;
```

Comment:
If you want to hoist this to the top level you should do it in the docs too

```suggestion
/// use icu_locid_transform::LocaleFallbacker;
```